app/tests/team_member_moods/test_team_member_moods_resource.py

```diff
@@ -0,0 +1,180 @@
+    def create_app(self):
+        return create_app("test")
+
+    def setUp(self):
+        upgrade(x_arg="data=true")
+        self.mood_template = create_mood_template_in_db("test_template")
+        self.mood1 = create_mood_in_db("mood1", "#ffffff", self.mood_template)
+        self.mood2 = create_mood_in_db("mood2", "#aaaaaa", self.mood_template)
+        self.test_team1 = create_team_in_db("test_team_name")
+        self.test_team2 = create_team_in_db("test_team_name2")
+        self.test_user1 = create_user_in_db("test_user1@test.com", "password")
+        self.test_user2 = create_user_in_db("test_user2@test.com", "password")
+        self.member_role = get_team_role_by_name("Member")
+        self.test_team_member_1 = create_team_member_in_db(
+            self.test_team1, self.test_user1, self.member_role
+        )
+        self.test_team_member_2 = create_team_member_in_db(
+            self.test_team1, self.test_user2, self.member_role
+        )
+
+    def tearDown(self):
+        db.session.remove()
+        downgrade(x_arg="data=true", revision="base")
+
+    def test_submit_team_member_mood_success(self):
+        response = self.client.post(
+            "/teams/"
+            + self.test_team1.public_id
+            + "/members/"
+            + self.test_team_member_1.public_id
+            + "/moods",
+            json={"mood_id": self.mood1.public_id},
+        )
+
+        json_response = response.get_json()
+
+        self.assertEqual(response.status_code, 201)
+        self.assertIn("id", json_response)
+        self.assertEqual(
+            json_response["team_member_id"], self.test_team_member_1.public_id
+        )
+        self.assertEqual(json_response["mood_id"], self.mood1.public_id)
+
+    def test_submit_team_member_mood_fails_with_invalid_team(self):
+        response = self.client.post(
+            "/teams/"
+            + "invalid_team_id"
+            + "/members/"
+            + self.test_team_member_1.public_id
+            + "/moods",
+            json={"mood_id": self.mood1.public_id},
+        )
+
+        json_response = response.get_json()
+
+        self.assertEqual(response.status_code, 404)
+        self.assertEqual(json_response["status"], "Failed")
+        self.assertEqual(
+            json_response["message"], "Team with id invalid_team_id does not exist"
+        )
+
+    def test_submit_team_member_mood_fails_with_invalid_mood(self):
+        response = self.client.post(
+            "/teams/"
+            + self.test_team1.public_id
+            + "/members/"
+            + self.test_team_member_1.public_id
+            + "/moods",
+            json={"mood_id": "invalid_mood_id"},
+        )
+
+        json_response = response.get_json()
+
+        self.assertEqual(response.status_code, 404)
+        self.assertEqual(json_response["status"], "Failed")
+        self.assertEqual(
+            json_response["message"], "Mood with id invalid_mood_id does not exist"
```

Comment:
Would be good to use f strings here and stick the invalid_id in a local variable.
mood_id = "invalid_mood_id"
f"Mood with id {mood_id} does not exist"